lib/lib.go

```diff
@@ -743,3 +744,20 @@ func (inst *Instance) RemoteClient() remote.Client {
 func (inst *Instance) Teardown() {
 	inst.teardown()
 }
+
+// checkRPCError validates RPC errors and in case of EOF returns a
+// more user friendly message
+func checkRPCError(err error) error {
+	if strings.Contains(err.Error(), "EOF") {
```

Comment:
err will often be `nil` here when it works, and will panic when the `Error()` method is called on a nil value. Adding this above will fix:

```go
if err == nil {
  return nil
}
```

Reply:
very sad to see this didn't crash the test suite anywhere.

Reply:
Yeah, thing is I thought adding a whole test suite for the RPCs against a running qri node. But thats quite a substantial bit of work and not really sure if it's worth the time if we're going to move away from it anyways.

Reply:
completely agreed. I'd _much_ rather see that energy go into refactoring our api package tests